tests/build-conf/beagleboneblack/local.conf

```diff
@@ -301,3 +301,6 @@ EXTERNALSRC_SYMLINKS = ""
 
 EXTRA_IMAGE_FEATURES:append:beaglebone-yocto = " ssh-server-openssh"
 IMAGE_INSTALL:append:beaglebone-yocto = " sudo"
+
+MENDER_STORAGE_TOTAL_SIZE_MB_DEFAULT ?= "608"
+IMAGE_OVERHEAD_FACTOR = "1.0"
```

Comment:
Could these be moved to[ `meta-mender-ci` instead](https://github.com/mendersoftware/meta-mender/blob/master/tests/meta-mender-ci/conf/layer.conf)? That way we won't have the values duplicated for all platforms.

Reply:
Ah, didn't know that this exists, thats definitely a better solution. As its already merged, want me to do a follow-up PR @lluiscampos?

Reply:
I think it cannot, because meta-mender-ci is not used for the Docker container, and it's needed there too.

Reply:
Yeah, the merge happened between I started to type and I finished my message :rofl: 

I think is best to follow-up :pray:

Reply:
You are right. Please don't follow-up :sweat_smile: